book/src/roadmap/2021.md

```diff
@@ -0,0 +1,235 @@
+# Roadmap 2021
```

Comment:
The roadmap/README.md file is currently empty, and we should also add some text there. I'll still have to think about something though.